InscryptionAPI/Triggers/CustomGlobalTriggerHandler.cs

```diff
@@ -0,0 +1,230 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using DiskCardGame;
+using System.Reflection;
+using System.Collections;
+using InscryptionAPI.Card;
+
+namespace InscryptionAPI.Triggers
+{
+    public static class CustomGlobalTriggerHandler
```

Comment:
I don't understand why this was added when we already have CustomTriggerFinder. If there is any functionality you need that's not in CustomTriggerFinder, add it there. You're also duplicating a lot of code that's in CustomTriggerFinder.

This class needs to be removed.

Reply:
should be done